draft-ietf-mls-architecture.md

```diff
@@ -1430,14 +1430,14 @@ deniability requires further analysis.
 
 ### Associating a User's Clients
 
-When the same user uses multiple clients, it may be possible for other members
-of a group to recognize all of those clients as belonging to the same user.  For
-example, all of a user's clients might present credentials authenticating the
-user's identity.  This association among devices might be considered a leak of
-private information.  The remainder of this section describes several approaches
-for addressing this.
+When a user has multiple devices, the base MLS protocol requires each device to
+be represented by a distinct client in the MLS groups that the user is a member
```

Comment:
I don't think this statement is correct, because as noted below it is possible to share state.

Reply:
It's correct. Key word is "base protocol". It's possible, but you need to build a lot of extra machinery to make this secure

Reply:
These are different statements. The base protocol doesn't have any facilities for state synchronization. That does not mean that it *requires* each device to be represented by a distinct client, any more than the fact TLS doesn't have any facility for transporting keys and certificates requires each TLS server to have a distinct certificate.